amoro-ams/src/main/java/org/apache/amoro/server/table/DefaultTableService.java

```diff
@@ -275,18 +274,24 @@ public void exploreExternalCatalog(ExternalCatalog externalCatalog) {
               try {
                 tableIdentifiersFutures.add(
                     CompletableFuture.supplyAsync(
-                        () -> {
-                          try {
-                            return externalCatalog.listTables(database).stream()
-                                .map(TableIdentity::new)
-                                .collect(Collectors.toSet());
-                          } catch (Exception e) {
-                            LOG.error(
-                                "TableExplorer list tables in database {} error", database, e);
-                            return new HashSet<>();
-                          }
-                        },
-                        tableExplorerExecutors));
+                            () -> {
+                              try {
+                                return externalCatalog.listTables(database).stream()
+                                    .map(TableIdentity::new)
+                                    .collect(Collectors.toSet());
+                              } catch (Exception e) {
```

Comment:
Hi，can we remove this try-catch here as we already handled the exception with the `exceptionally ` call?